app/addons/replication/actions.js

```diff
@@ -171,13 +298,20 @@ const deleteDocs = (docs) => {
   });
 };
 
-const getReplicationStateFrom = (id) => {
-  $.ajax({
-    url: `${app.host}/_replicator/${encodeURIComponent(id)}`,
-    contentType: 'application/json',
-    dataType: 'json',
+export const getReplicationStateFrom = (id) => {
```

Comment:
No, that is correct. Its getting the replication state from a document with `id`